src/assets/wise5/authoringTool/node/node-authoring/node-authoring.component.ts

```diff
@@ -501,11 +469,11 @@ class NodeAuthoringController {
     });
   }
 
-  scrollToTopOfPage() {
-    this.$anchorScroll('top');
+  scrollToTopOfPage(): void {
```

Comment:
Add ```private``` to all functions that are only called from this class?
```suggestion
  private scrollToTopOfPage(): void {
```